frontend/src/app/modules/availability/new-slot/questions/questions.component.ts

```diff
@@ -0,0 +1,51 @@
+import { CdkDragDrop, moveItemInArray } from '@angular/cdk/drag-drop';
+import { Component, Input } from '@angular/core';
+import { getLocalMandatoryQuestions } from '@core/helpers/questions-mandatory-helper';
+import { IAvailabilitySlot } from '@core/models/IAvailiabilitySlot';
+import { IQuestion } from '@core/models/IQuestion';
+
+@Component({
+    selector: 'app-questions',
+    templateUrl: './questions.component.html',
+    styleUrls: ['./questions.component.sass'],
+})
+export class QuestionsComponent {
+    @Input() set newSlot(slot: IAvailabilitySlot | undefined) {
```

Comment:
```suggestion
    @Input() set newSlot(slot?: IAvailabilitySlot) {
```

Reply:
At some point I tried to use '?' but got this error and decided to move on.

A 'set' accessor cannot have an optional parameter.ts(1051) 
![image](https://user-images.githubusercontent.com/34366638/186925165-547480e0-85f5-4edf-86f1-4f4da7ac4700.png)